README.md

```diff
@@ -42,7 +42,15 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
+S -> A B A  
+A -> < empty >  
+A -> "r"  
+A -> "g"  
+A -> "b"  
+B -> S  
+B -> "r"  
+B -> "g"  
+B -> "b"  
```

Comment:
# 0

Sorry, but this is way off... I was looking for something like

~~~
       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b
~~~